### PR TITLE
Gutting functionality; replacing with static background

### PR DESCRIPTION
**Pull request by mintsoft**
The new version has resized the viewports to 188x120 which is too small
for the old functionality. Implemented to address https://github.com/duckduckgo/opera-speeddial/issues/1

### CODE REVIEW

**Comment by mintsoft** [comment]
image

File: manifest.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 "name": "DuckDuckGo - Speed Dial",
 "developer": {"name" : "DuckDuckGo", "url" : "http://duckduckgo.com/?t=oen"},
-"description" : "Shows random Zero-click Info from DuckDuckGo in a Speed Dial window.",
+"description" : "Shows DuckDuckGo in a Speed Dial window.",
 "manifest_version" : 2,
 "version" : "1.3.1",
 "background" : {"page" : "index.html"},
 "icons" : {"128": "images/icon_64.png"},
-"web_accessible_resources" : [".gitignore","images/icon_64.png","images/logo.svg","readme.md","js/jquery.min.js","js/background.js","style.css"],
+"web_accessible_resources" : [".gitignore","images/icon_64.png","images/logo.svg","images/small_background.png","images/large_background.png","readme.md","style.css"],
```
Comment (mrshu) [inline manifest.json]: @mintsoft I am not sure why `.gitignore` has to be here, but I guess it does not hurt...

**Comment by mrshu** [comment]
@mintsoft Thanks for doing this!

I'll test it tomorrow and then just merge it, there is not too much to discuss here =)

**Comment by mrshu** [comment]
@mintsoft So I actually found some time to try it out in Opera and it indeed looks good!

The only comment I'd have is that Dax seems to be a bit blurry to me, especially compared to say Facebook or Amazon (see attached -- please open the original image by clicking on it). 

@chrismorast can you see that too or is it just me?

**Comment by javathunderman** [comment]
Commenting for updates- nice to see this alive.

**Comment by mintsoft** [comment]
@mrshu It's probably because it's a PNG. @chrismorast can you chuck me over an SVG?

**Comment by chrismorast** [comment]
Sure, @mintsoft. Here you go. https://www.dropbox.com/s/rcwdvg4cbwkgwyk/DDG_1x.svg?dl=0

**Comment by mintsoft** [comment]
@mrshu Apologies for the delay, it should now be SVG & therefore blur free!

**Comment by mrshu** [comment]
@mintsoft Awesome, this looks great!

I'll build and deploy this updated version soon!

**Pull request merged**